specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/Watchlists.json

```diff
@@ -566,7 +566,7 @@
       },
       "required": [
```

Comment:
watchlistItemsCount was removed from stable version and should also be removed from this version.